### PR TITLE
Fixed translation of levels

### LINKED ISSUE #2420: (SP: ) Fix translation for levels on the filter drawer
Add Ukrainian translation for levels on the filter drawer
Screenshot 2024-09-02 at 21 00 30

### PR DESCRIPTION
**Pull request by Mav-Ivan**
image

### CODE REVIEW

**Review by BohdanMylyi — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

File: src/components/checkbox-list/CheckboxList.tsx
Comment (dudchakk) [inline src/components/checkbox-list/CheckboxList.tsx:1]: why did you remove proficiencyLevelLabels? it's conventional to use them for translation of proficiency levels in our project, as in these PRs:
https://github.com/ita-social-projects/SpaceToStudy-Client/pull/2292
https://github.com/ita-social-projects/SpaceToStudy-Client/pull/2474
https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1975

Comment (Mav-Ivan) [reply src/components/checkbox-list/CheckboxList.tsx:1]: The same values came from checkboxesList , and for some reason  proficiencyLevelLabels was undefined
